frontend/src/styles/AssignBtn.css

```diff
@@ -0,0 +1,136 @@
+.btn_assign {
```

Comment:
Most of these styles are carbon copies of the others, just some changes to background/text color.  I'd make `AssignBtn.js` return a button with a className that's `"btn_assign btn_[specific type here]"`, that way you can remove all the styles that aren't background, border, and color from all the classes except `.btn_assign`.